lib/oxidized/model/edgecos.rb

```diff
@@ -24,6 +28,8 @@ class EdgeCOS < Oxidized::Model
   end
 
   cmd 'show watchdog' do |cfg|
+    # Do not show error as command is not supported on some devices
+    cfg.gsub! /^(% Invalid input detected at '\^' marker\.|^\s+\^)$/, ''
```

Comment:
Should this perhaps be handled under `cmd :all` to suppress all errors regardless of where they occur? You can find an example in `fastiron.rb`, another model which targets devices where command availability varies.

Reply:
I don't know. I think it is better to selectively suppress errors only for those commands where we expect them.

Reply:
Do we know that the watchdog command specifically is omitted in some versions, while the other commands are universal across EdgeCOS versions? I don't have a strong preference here, but dealing with this in `:all` has the added benefit of not finding ourselves in the future in a situation where one contributor running one version of the model adds a command, for another user who runs a different version to discover the command does not exist there. 

There are upsides and downsides to either approach, I'll leave this at your discretion.

Reply:
I've made the change as you suggested.